ui/StatusQ/src/StatusQ/Popups/Dialog/StatusDialog.qml

```diff
@@ -53,6 +53,12 @@ Dialog {
     */
     property bool fillHeightOnBottomSheet: false
 
+    /*!
```

Comment:
Wondering why we need a new property `fullScreen` and not reuse the `fillHeightOnBottomSheet`?

Reply:
Also having some pages on total `fullScreen` if they don't have the close button, how to interact with them (close or go back)? I.e: `Chat search`

Reply:
Yup, works only with the default `header`

Reply:
Yes, exactly! Works only with the default header and when there's a close button. We need to make sure the user doesn't get stuck and the popup can be dismissed.

Now why a new full screen property and not reusing the fill height property? The fill height is a fallback for all the other popups we want to be full screen, but won't work nicely in full screen mode for different reasons. E.g. send, swap